app/src/main/java/android/coding/interview/makeitawesome/adapter/PhotosAdapter.java

```diff
@@ -35,11 +64,34 @@ public int getItemCount() {
         return mData.size();
     }
 
-    public static class ViewHolder extends RecyclerView.ViewHolder {
-        final TextView mText;
+    public class ViewHolder extends RecyclerView.ViewHolder implements View.OnClickListener {
```

Comment:
what is the possible implications of having non-static (i.e. inner local) `ViewHolder` class ?